### ISSUE TITLE
Layer Normalization from pytorch

### ISSUE THREAD

**Issue by edmBernard** [megvii-research/NAFNet#148]
I see that you implement yourself the layer normalization instead of using the LayerNorm available in Pytorch
https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html

Is there a reason to not use the one from Pytorch ? 

Thanks
